packages/vscode-extension/src/debug/depsChecker/doctorConstant.ts

```diff
@@ -19,6 +19,8 @@ export const doctorConstant = {
   NodeNotFound: `Cannot find Node.js. ${InstallNode}`,
   NodeNotSupported: `Node.js (@CurrentVersion) is not in the supported version list (@SupportedVersions). ${InstallNode}`,
   NodeSuccess: `Supported Node.js version (@Version) is installed`,
+  BypassNode12: `To continue local debugging with Node.js v12, please go to Visual Studio Code setting page for Teams Toolkit and uncheck Node.js.`,
+  BypassNode12AndFunction: `To continue local debugging with Node.js v12, please go to Visual Studio Code setting page for Teams Toolkit and uncheck Node.js and Azure Function Core Tools. And use your Azure Function Core Tools V3.`,
```

Comment:
To continue to local debug using Node.js v12, go to Visual Studio Code Settings, under Teams Toolkit, Prerequisite Check, uncheck "Ensure Node.js is installed. (node)" and "Ensure Azure Function Core Tools is installed. (funcCoreTools)". Also make sure you install Azure Function Core Tools v3 https://github.com/Azure/azure-functions-core-tools.